react-accordion-component/my-app/src/Accordion.js

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import './Accordion.css';
+
+export default function Accordion({ languages }) {
+  const [clicked, setClicked] = useState('');
+
+  function handleLanguageClick(language) {
+    clicked === language ? setClicked('') : setClicked(language);
+  }
+
+  return (
+    <Languages
+      languages={languages}
+      handleOnClick={handleLanguageClick}
+      clicked={clicked}
+    />
+  );
+}
+
+function Languages({ languages, handleOnClick, clicked }) {
+  function renderAbout(language) {
+    return clicked === language ? <p>{language.about}</p> : null;
+  }
+
+  return (
+    <div className="language-tab">
+      {languages.map((language) => {
+        return (
+          <div key={language.language}>
+            <button
+              className="names"
+              type="button"
+              onClick={(event) => handleOnClick(language)}>
+              {language.language}
+            </button>
+            {renderAbout(language)}
```

Comment:
It's more common to just inline this instead of using a separate function call. The best usage would be:
`{clicked === language && <p>{language.about}</p>}`